src/screens/TextInstructionScreen.tsx

```diff
@@ -13,35 +18,50 @@ export const TextInstructionScreen: React.FunctionComponent<TextInstructionScree
   description,
   actionLabel,
   color = 'purple',
+  textColor = 'darkGrey',
+  backgroundColor = 'white',
+  linkColor = 'white',
+  textAlign = 'center',
   onNext,
 }) => {
   return (
-    <Box
-      flex={1}
-      alignItems="center"
-      justifyContent="flex-start"
-      pt={24}
-      px={5}
-    >
-      <Text variant="instructionHeading" mb={5}>
-        {heading}
-      </Text>
-      <Text variant="instructionDescription" mb={10}>
-        {description}
-      </Text>
-      {actionLabel && (
-        <Text variant="instructionActionLabel" px={6}>
-          {actionLabel}
+    <SafeAreaView flex={1} backgroundColor={backgroundColor}>
+      <Box
+        flex={1}
+        alignItems="center"
+        justifyContent="flex-start"
+        backgroundColor={backgroundColor}
+        pt={10}
+        px={5}
+      >
+        <Text variant="instructionHeading" mb={5} color={textColor}>
+          {heading}
         </Text>
-      )}
-      <Box flex={1} justifyContent="flex-end" pb={14}>
-        <Button
-          variant="primary"
-          label="Next"
-          backgroundColor={color}
-          onPress={onNext}
-        />
+        <Text
+          variant="instructionDescription"
+          mb={10}
+          color={textColor}
+          textAlign={textAlign}
+        >
+          {description}
```

Comment:
Needs to be markdown?

Reply:
Yes but it has wider implications so I'll handle this on the integration part.